unlock-app/src/__tests__/hooks/useLocks.test.js

```diff
@@ -239,11 +239,10 @@ describe('useLocks', () => {
       })
     })
 
-    const { result, waitForNextUpdate } = renderHook(() =>
-      useLocks(ownerAddress)
-    )
-    await waitForNextUpdate() // First render when lock is loaded through The Graph
-    await waitForNextUpdate() // Second render when the lock is loaded through past transactions
+    const { result, wait } = renderHook(() => useLocks(ownerAddress))
+    await wait(() => {
```

Comment:
instead of waiting for a number of "renders" which could actually differ (given that the the `useReducer` callback will only trigger a re-render if the data differs and the data may be different based on the timing of the data retrievals), we wait for loading to be done, which will always happen!